fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/RecordConstructorValue.java

```diff
@@ -140,16 +140,11 @@ public int hashCode() {
 
     @Nonnull
     @Override
-    public String describe(@Nonnull final Formatter formatter) {
+    public String explain(@Nonnull final Formatter formatter) {
         return "(" +
                childrenAndNames.stream()
-                       .map(childAndName -> {
-                           if (childAndName.getValue().isPresent()) {
-                               return childAndName.getKey().describe(formatter) + " as " + childAndName.getValue().get();
-                           }
-                           return childAndName.getKey().describe(formatter);
-                       })
-                       .collect(Collectors.joining(", ")) + ")";
+                       .map(childAndName -> childAndName.getKey().explain(formatter))
+                       .collect(Collectors.joining(", ")) + ") as Record";
```

Comment:
I think you are dropping the name of the field here.

I think we should not do the `as record` (similar to the comment I made on `as array` further up). This code is e.g. used to explain a `SelectExpression`. That would now look like:
```
SELECT (x + 5, y + 3) as record
```

which I think is not as good as

```
SELECT (x + 5 as a, y + 3 as b)
```

the `as record` also leads people to believe this is an alias naming which in this case it is not.

Reply:
Thanks for the great observation, I will change it.